controller_manager/src/controller_manager.cpp

```diff
@@ -29,14 +29,16 @@ namespace controller_manager
 static constexpr const char * kControllerInterfaceName = "controller_interface";
 static constexpr const char * kControllerInterface = "controller_interface::ControllerInterface";
 
-inline bool is_controller_inactive(const controller_interface::ControllerInterface & controller)
+inline bool is_controller_inactive(controller_interface::ControllerInterface & controller)
```

Comment:
https://www.youtube.com/watch?v=ztABYog5x3o

Reply:
So for lifecycle nodes the compiler was unhappy with this being const.

Reply:
Now it is happy so its back